Formula/s/spotbugs.rb

```diff
@@ -5,6 +5,11 @@ class Spotbugs < Formula
   sha256 "11629b13aad39c453c23f8a8a43096b003afb55924a17424a9e1bc722190576b"
   license "LGPL-2.1-or-later"
 
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url "https://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs/"
    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
```
Something like his would be better to align livecheck with the location of the stable source.